scripts/automation/Radius/Distribute-UserCerts.ps1

```diff
@@ -2,74 +2,78 @@
 . "$psscriptroot/config.ps1"
 Connect-JCOnline $JCAPIKEY -force
 
-# Functions
-function New-JCCommandFile {
-    [CmdletBinding()]
-    param (
-        [Parameter(Mandatory = $true)][System.IO.FileInfo]$certFilePath,
-        [Parameter(Mandatory = $true)][String]$FileName,
-        [Parameter(Mandatory = $true)][String]$FileDestination
-    )
-    begin {
-        $headers = @{
-            "x-api-key" = $JCAPIKEY
-            "x-org-id"  = $JCORGID
-        }
-        $body = @{
-            content     = [convert]::ToBase64String((Get-Content -Path $certFilePath -AsByteStream))
-            name        = $FileName
-            destination = $FileDestination
+################################################################################
+# Do not modify below
+################################################################################
+
+# Import the functions
+Import-Module "$psscriptroot/RadiusCertFunctions.ps1" -Force
+
+# Import the users.json file and convert to PSObject
+$userArray = Get-Content -Raw -Path "$PSScriptRoot/users.json" | ConvertFrom-Json -Depth 6
+# Check to see if previous commands exist
+$RadiusCertCommands = Get-JCCommand | Where-Object { $_.Name -like 'RadiusCert-Install*' }
+
+if ($RadiusCertCommands.Count -ge 1) {
+    Write-Host "[status] $([char]0x1b)[96mRadiusCert commands detected, please make a selection."
+    Write-Host "1: Press '1' to generate new commands for ALL users. $([char]0x1b)[96mNOTE: This will remove any existing commands"
```

Comment:
I had to go look at the code before I ran this, I was a little spooked it would "remove any existing commands"